js/Signup.js

```diff
@@ -92,15 +92,24 @@ export default class Signup extends Component {
   }
 
   _submit() {
-    // console.warn(this.props, 'props');
+    // const url1 = 'http://ec2-34-238-240-14.compute-1.amazonaws.com/signup';
+    const testServer = 'http://192.168.0.15:8200/signup';
     axios({
       method: 'post',
-      url: 'http://ec2-34-238-240-14.compute-1.amazonaws.com/login',
-      data: this.state,
+      url: testServer,
+      data: {
+        username: this.state.username,
+        email: this.state.email,
+        password: this.state.password,
+      },
     })
       .then((response) => {
-        // console.warn('response', response);
-        this.props._logIn();
+        console.warn(123456789, response);
```

Comment:
console warn. Do we still need this? is this still in progress?

Reply:
I had been working on Login and forget to clean up Signup. we do not need this. Signup and Login are completely finsihed, minus tokens and refresh.